source/_docs/configuration/packages.markdown

```diff
@@ -74,15 +74,15 @@ Integrations inside packages can only specify platform entries using configurati
 
 ## Create a packages folder
 
-One way to organize packages is to create a folder named "packages" in your Home Assistant configuration directory. In the packages directory, you can store any number of packages in a {% term YAML %} file. This entry in your {% term "`configuration.yaml`" %} will load all {% term YAML %}-files in this _packages_ folder and its subfolders:
+One way to organize packages is to create a folder named "packages" in your Home Assistant configuration directory. In this packages folder you can store any number of packages in {% term YAML %} files, and organise those packages into {% term YAML %}-files and subfolders as you see fit. With `!include_dir_named` the file name is used as the package name. This means that file names must be globally unique, even across sub-folders. This entry in your {% term "`configuration.yaml`" %} will load all {% term YAML %}-files in this _packages_ folder and its subfolders:
```

Comment:
suggested changes for grammar and consistency and inline with documentation to be in American English:
```
One way to organize packages is to create a folder named "packages" in your Home Assistant configuration directory. In this packages folder, you can store any number of packages in {% term YAML %} files, and organize those packages into {% term YAML %}-files and sub folders as you see fit. With `!include_dir_named` the file name is used as the package name. This means that file names must be globally unique, even across sub folders. This entry in your {% term "`configuration.yaml`" %} will load all {% term YAML %}-files in this _packages_ folder and its sub folders:
```

Reply:
Thanks, these look good, I have applied.